static/docs/user-guide/install/completion.md

```diff
@@ -0,0 +1,154 @@
+
+Depending on the shell that you are using, follow the instruction below to
+install and configure shell completion.
+
+## Bash completion on Mac OS
+
+1. First, make sure that Bash completion support is installed:
+
+   ```dvc
+   $ brew install bash-completion
+   ```
+
+2. Then download the
+   [DVC completion script](https://github.com/iterative/dvc/blob/master/scripts/completion/dvc.bash)
+   and save it on `/usr/local/etc/bash_completion.d/`:
+
+   ```dvc
+   $ sudo wget \
+       -O /usr/local/etc/bash_completion.d/dvc \
+       https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.bash
+   ```
+
+3. Edit `~/.bash_profile` and make sure that these lines are there:
+
+   ```bash
+   if [ -f $(brew --prefix)/etc/bash_completion ]; then
+       . $(brew --prefix)/etc/bash_completion
+   fi
+   ```
+
+4. Finally, `source ~/.bash_profile` or launch a new terminal to activate it.
+
+## Bash completion on Ubuntu
+
+On a Linux system bash-completion should work out of the box, but if you are
+using a VPS on the cloud it might not be installed. In any case, follow these
+steps to make sure that it is installed:
+
+1. First, make sure that Bash completion support is installed:
+
+   ```dvc
+   $ sudo apt install --reinstall bash-completion
+   ```
+
+2. Then download the
+   [DVC completion script](https://github.com/iterative/dvc/blob/master/scripts/completion/dvc.bash)
+   and save it on `/etc/bash_completion.d/`:
+
+   ```dvc
+   $ sudo wget \
+       -O /etc/bash_completion.d/dvc \
+       https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.bash
+   ```
+
+3. Edit `~/.bashrc` and make sure that these lines are there:
+
+   ```bash
+   # enable bash completion in interactive shells
+   if ! shopt -oq posix; then
+     if [ -f /usr/share/bash-completion/bash_completion ]; then
+       . /usr/share/bash-completion/bash_completion
+     elif [ -f /etc/bash_completion ]; then
+       . /etc/bash_completion
+     fi
+   fi
+   ```
+
+4. Finally, `source ~/.bashrc` or open a new terminal to activate it.
+
+For more details see: https://linuxhandbook.com/enable-tab-completion/
+
+## Zsh completion
+
+The DVC specific completion script is located in this path of our main
+repository:
+[dvc/scripts/completion/dvc.zsh](https://github.com/iterative/dvc/blob/master/scripts/completion/dvc.zsh)
+
+Place the completion script in a directory included in `$fpath`, the file name
+should be `_dvc`.
+
+For example:
+
+```dvc
+$ sudo wget \
+    -O /usr/share/zsh/site-functions/_dvc \
+    https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.zsh
+```
+
+Make sure `compinit` is loaded or do it by adding in `~/.zshrc`:
+
+```dvc
+$ autoload -Uz compinit && compinit -i
+```
+
+Then reload your shell:
+
+```dvc
+$ exec $SHELL -l
+```
+
+This step is optional but will make the DVC output look much nicer, by adding
+more colors to it. Add the following to your `~/.zshrc`:
+
+```bash
```

Comment:
is there zsh?

Reply:
There is, but the result is the same. Anyway fixed it.

Reply:
The generic one is `sh`